rke/k8s_rke_system_images.go

```diff
@@ -5388,10 +5388,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
-		// Enabled out of band post v2.5.9
-		"v1.19.14-rancher1-1": {
+		// Enabled in v2.6.1
```

Comment:
for v1.19.15-rancher1-1, it should be `Enabled out of band post v2.5.x`

Reply:
Due to the OOB release, this comment will need to be `Enabled out of band post v2.5.9`, which is the release it was introduced after.